app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -542,19 +542,36 @@
   </div>
   <br>
   <div class="{{unless device.isMobile 'right floated large' 'mini three'}} ui fields buttons">
```

Comment:
@uds5501 Sorry, didn't notice it due to the long line and you are rerendering all of them--- but this is just too much redundancy. You can just make this line 
```hbs
 <div class="{{unless device.isMobile 'right floated large' 'mini three'}} ui fields {{if isUserUnverified 'disabled' buttons">
```
 and it will have the same effect without an else clause.

Reply:
@CosmicCoder96  This isn't working :sweat_smile:

Reply:
I didn't close the curly brace of if. Did you ?

Reply:
I did

Reply:
Push your non working code with this fix. @uds5501

Reply:
Understood

Reply:
@CosmicCoder96  Did it